src/network/networkd-manager.h

```diff
@@ -39,6 +39,7 @@ struct Manager {
         LinkOperationalState operational_state;
         LinkCarrierState carrier_state;
         LinkAddressState address_state;
+        LinkOnlineState online_state;
```

Comment:
This should be also initialized with `_LINK_ONLINE_STATE_INVALID` in `manager_new()`.